.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: scala
+scala:
+  - 2.11.8
+  - 2.10.6
+jdk:
+  - oraclejdk8
+
+script:
+  - sbt clean coverage test coverageReport
+after_success:
+  - sbt coveralls
```

Comment:
We should use codecov.io and switch this to match: https://github.com/codecov/example-scala/blob/master/.travis.yml